Source/WebCore/css/CSSProperties.json

```diff
@@ -9459,51 +9210,6 @@
                 "parser-grammar": "<<values>>"
             },
             "status": "non-standard"
-        },
-        "fill-color": {
```

Comment:
Let's file a supportive position on the Fill and Stroke module.

Reply:
https://github.com/WebKit/standards-positions/issues/202